contracts/Bidder.sol

```diff
@@ -28,13 +27,14 @@ contract Bidder is SimpleFlopper {
   function submitBid() external {
     require(msg.sender == owner, "Bidder/submitBid: owner only");
 
+    // dai has 45 decimal places
     (uint256 amountDai, , , , ) = getCurrentBid(bid);
 
-    amountBid = amountDai;
-    expectedLot = amountDai / 100;
+    // lot needs to have 18 decimal places, and we're expecting 1 mkr = 100 dai
+    expectedLot = amountDai / 1e27 / 100;
```

Comment:
Just to clarify, is this `expectedLot = (amountDai / 1e27) / 100;` or `expectedLot = amountDai / (1e27 / 100);`?